tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -314,8 +314,10 @@ def create_parser(product_choices=None):
 
     servo_group = parser.add_argument_group("Chrome-specific")
     servo_group.add_argument("--enable-mojojs", action="store_true", default=False,
-                             help="Enable MojoJS for testing. Mojo bindings need to be available in "
-                             "_venv2/mojojs.")
+                             help="Enable MojoJS for testing")
```

Comment:
General question/nit: it seems like there's some non-trivial interaction among these flags (and other things like browser channel) that make them a little unpredictable. Would it be possible to make the code dumber and not set anything implicitly?

I'm thinking:
- running with no mojo flags should not download mojojs and not install it, meaning mojo-based test would fail
- running with --mojojs-path is only for locally-built mojo (ie: chromium ci)
- running with --enable-mojojs is only for downloading mojo (so maybe rename to --download-mojojs)

There may be good reasons not to do this, but just wondering :)

Reply:
Specifically I want the default behaviour (without Mojo-related flags) to enable MojoJS, which then implies automatic downloading. The rationale is that this is the same as disabling popup blocker, ignoring cert errors, etc., something required to run tests.

Reply:
Ok fair enough.
I still worry that it's hard to understand how to set these by just reading the help strings (which is true for many flags, but I feel like we shouldn't make it worse). Although I guess the auto-downloading part will be simpler once Chrome 86 is stable?

So I guess it might be nice to say "Mojo bindings must be available" here, and mention that you can either use --mojojs-path or they will auto download. This is a nit.

Reply:
Yeah Chrome 86 will be stable next week :)

I'll tweak the help message to reflect the auto downloading.

Reply:
```suggestion
                             help="Enable MojoJS for testing. Note that this flag is usally "
                             "enabled automatically by `wpt run`, if it succeeds in downloading "
                             "the right version of mojojs.zip or if --mojojs-path is specified.")
```